File: Cpp/168_Excel_Sheet_Column_Title.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string result = "";
        int current = n;
        while(current){
            --current;
            result += current%26 + 'A';
            current /= 26;
        }
        
        reverse(result.begin(), result.end());
        
        return result;
    }
};

File: Kotlin/168_Excel_Sheet_Column_Title.kt
class Solution {
    fun convertToTitle(columnNumber: Int): String {
        val result = StringBuilder()
        var current = columnNumber
        
        while (current > 0) {
            --current
            result.append((current % 26 + 'A'.toInt()).toChar())
            current /= 26
        }
        
        return result.toString().reversed()
    }
}